src/main/java/org/apache/lucene/analysis/kr/utils/DictionaryUtil.java

```diff
@@ -312,12 +296,12 @@ public static String combineAndEomiCheck(char s, String eomi) throws MorphExcept
 	 * @param type	1: josa, 2: eomi
 	 * @throws MorphException
```

Comment:
For the guava issue: yep, 18.0 but how is ES core depending on Guava 18.0? It's listed in their gradle build file as a test dependency only